platform/app/cypress/integration/measurement-tracking/OHIFDownloadSnapshotFile.spec.js

```diff
@@ -36,6 +36,8 @@ describe('OHIF Download Snapshot File', () => {
     // Check buttons
     cy.get('[data-cy="cancel-btn"]').scrollIntoView().should('be.visible');
     cy.get('[data-cy="download-btn"]').scrollIntoView().should('be.visible');
+
+    cy.get('[data-cy="cancel-btn"]').click();
```

Comment:
The close was throwing an exception, and without it, this test passes, but the next one fails.